build-system/cpp-cmake-conan/src/build.py

```diff
@@ -172,6 +158,11 @@ def cli() -> None:
     parser.add_argument(
         "-s", "--static", action="store_true", help="Links all dependencies statically."
     )
+    parser.add_argument(
+        "--toolchain",
```

Comment:
Where do we want to document how to use this in more detail, as it is not obvious to me. Like what is expected to exist in the file that shall be referenced. Long run I think it would be good to have a more hands on example.

Is this supposed to be documented here, or possibly in https://github.com/eclipse-velocitas/velocitas-docs, or only in the template supposed to use it.

Reply:
Documentation added to https://github.com/SoftwareDefinedVehicle/vehicle-app-cpp-template-lattice/pull/59

I think that is good enough for now, but long term we may need to discuss if we want to have velocitas-docs as the primary documentation source or rather markdown in the templates.